File: HealthSysHub.Web.Utility/Models/HospitalSpecialtyInformation.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace HealthSysHub.Web.Utility.Models
{
    public class HospitalSpecialtyInformation
    {
        public Guid? HospitalSpecialtyId { get; set; }
        public Guid? HospitalId { get; set; }
        public Guid? SpecializationId { get; set; }
        public string? SpecializationName { get; set; }
        public Guid? CreatedBy { get; set; }
        public DateTimeOffset CreatedOn { get; set; }
        public Guid? ModifiedBy { get; set; }
        public DateTimeOffset? ModifiedOn { get; set; }
        public bool IsActive { get; set; }
    }
}

File: HealthSysHub.Web.Pharmacy.UI/src/app/models/hospital-specialty-information.ts
export interface HospitalSpecialtyInformation {
  hospitalSpecialtyId?: string | null;
  hospitalId?: string | null;
  specializationId?: string | null;
  specializationName?: string | null;
  createdBy?: string | null;
  createdOn: Date | string;
  modifiedBy?: string | null;
  modifiedOn?: Date | string | null;
  isActive: boolean;
}
